.github/workflows/ci-windows.yml

```diff
@@ -86,17 +81,37 @@ jobs:
           key: ${{ env.CACHE_KEY }}
           restore-keys: windows-build-test-cpp-
 
+      - name: Hack IREE
+        shell: bash
+        run: |
+          # TODO(max): send IREE a fix for this
+          # target_compile_definitions may only set INTERFACE properties on IMPORTED
+          sed -i '114s/PRIVATE/INTERFACE/g' third_party/iree/compiler/src/iree/compiler/API/CMakeLists.txt
```

Comment:
👀 
Is 114 a line number, pointing here: https://github.com/iree-org/iree/blob/45132f5b1f1697e1f4d3450f0d6371c2ec49700a/compiler/src/iree/compiler/API/CMakeLists.txt#L114? At a minimum it would be nice to make that more stable and not reliant on the file remaining unchanged.

I haven't noticed that "target_compile_definitions may only set INTERFACE properties on IMPORTED" error/warning upstream. Seems reasonable enough to patch it if it is causing issues downstream.

Reply:
yea that's the line - i'm guessing no one ever exercises the combo of "windows" + "byo llvm".